src/components/biggive-table/biggive-table.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-table',
+  styleUrl: 'biggive-table.scss',
+  shadow: false,
+})
+export class BiggiveTable {
+  /**
+   * Space below component
+   */
```

Comment:
We talked on https://github.com/thebiggive/components/pull/98 about only using docblocks where there's something to add beyond what the code says. Is there a reason for the docblocks on these props?